src/pages/patientView/mutation/column/ProteinChangeColumnFormatter.tsx

```diff
@@ -1,39 +0,0 @@
-import * as React from 'react';
-import {
-    default as DefaultProteinChangeColumnFormatter
-} from "shared/components/mutationTable/column/ProteinChangeColumnFormatter";
-import MutationStatusColumnFormatter from "shared/components/mutationTable/column/MutationStatusColumnFormatter";
-import {Mutation} from "shared/api/generated/CBioPortalAPI";
-import styles from './style/proteinChange.module.scss';
-
-/**
- * Designed to customize protein change column content for patient view page.
- *
- * @author Selcuk Onur Sumer
- */
-export default class ProteinChangeColumnFormatter
```

Comment:
Moving this renderer into the default protein change column formatter, since we want this behavior to be default now